intelligence-services/latest/admin/index.md

```diff
@@ -153,3 +154,37 @@ See the AWS site for more details on service limits: [Limits in Amazon Textract]
 You'll need to create an AWS Identity and Access Management (IAM) role with the correct permissions to control access to AWS services and resources.
 
 There's a setting for the level of confidence that Amazon Textract has in the accuracy of the extracted content. This is defined as the minimum confidence level and has a default value of 80%.
+
+### Amazon Transcribe
+
+Amazon Transcribe makes it easy for you to generate speech to text from your video and audioo files.
+
+This service can detect text in a variety of documents (such as financial reports, medical records, and tax forms). For documents with structured data, the following can be detected:
+
+* Forms with their fields and values
+* Tables with their cells
+
+#### Prerequisites (Textract)
+
+The general prerequisites to use Amazon Textract are documented in [Getting Started with Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/getting-started.html){:target="_blank"}.
+
+#### Supported regions (Textract)
+
+See the list of supported AWS regions where Amazon Textract is [available](https://docs.aws.amazon.com/general/latest/gr/rande.html){:target="_blank"}.
+
+#### Limits (Textract)
+
+There are a number of limits that relate to Amazon Textract:
+
+* Amazon Textract synchronous operations (`DetectDocumentText` and `AnalyzeDocument`) support the PNG and JPEG image formats. The maximum document image (JPG/PNG) size is 5 MB.
+* Asynchronous operations (`StartDocumentTextDetection`, `StartDocumentAnalysis`) also support the PDF file format. The maximum PDF file size is 500 MB, and a maximum of 3000 pages.
+  * To process PDF documents, we use asynchronous operations that go via an S3 bucket setup for Intelligence Services and Textract.
+  * The maximum number of concurrent jobs for all asynchronous operations is 1.
+
+See the AWS site for more details on service limits: [Limits in Amazon Textract](https://docs.aws.amazon.com/textract/latest/dg/limits.html){:target="_blank"}.
+
+#### Configuration (Textract)
+
+You'll need to create an AWS Identity and Access Management (IAM) role with the correct permissions to control access to AWS services and resources.
+
+There's a setting for the level of confidence that Amazon Textract has in the accuracy of the extracted content. This is defined as the minimum confidence level and has a default value of 80%.
```

Comment:
This whole block is referring to Textract instead of transcribe. Also, links should point to transcribe (https://docs.aws.amazon.com/transcribe/latest/dg/getting-started.html)

Reply:
Also you can refer to https://docs.aws.amazon.com/transcribe/latest/dg/limits-guidelines.html for the limits the services you are mentioning do not belong to transcribe

Reply:
Done.